sotopia/cli/install/install.py

```diff
@@ -313,8 +314,11 @@ def install(
                     )
                     exit(0)
             else:
-                (Path(tmpdir) / "dump.rdb").rename(
-                    Path(directory) / "redis-data/dump.rdb"
+                # (Path(tmpdir) / "dump.rdb").rename(
+                #     Path(directory) / "redis-data/dump.rdb"
+                # )
+                shutil.move(
```

Comment:
Is it necessary to change this?

Reply:
`shutil.move()` can move files across different file systems. The commented-out code will throw an error on my computer.

Reply:
cool, maybe remove the commented lines then